docs/reference/yaml.md

```diff
@@ -0,0 +1,132 @@
+# YAML format reference
```

Comment:
Secrets are missing

Reply:
Secrets may be covered elsewhere, so let's mention them here and the format then link to that other page too.